xcodeproject/xcscheme/xcscheme.go

```diff
@@ -92,6 +162,58 @@ func Open(pth string) (Scheme, error) {
 	return scheme, nil
 }
 
+// XMLToken ...
+type XMLToken int
+
+const (
+	invalid XMLToken = iota
+	// XMLStart ...
+	XMLStart
+	// XMLEnd ...
+	XMLEnd
+	// XMLAttribute ...
+	XMLAttribute
+)
+
+// Marshal ...
+func (s Scheme) Marshal() ([]byte, error) {
+	contents, err := xml.Marshal(s)
+	if err != nil {
+		return nil, fmt.Errorf("failed to marshal Scheme: %v", err)
+	}
+
+	contentsNewline := strings.ReplaceAll(string(contents), "><", ">\n<")
```

Comment:
What about "> <" or there is no such thing?

Reply:
It could be valid, but it is not generated by the go xml package (in the unformatted output).